### PR TITLE
Consider todos

### PR DESCRIPTION
**Pull request by Anaphory**

### CODE REVIEW

**Comment by Anaphory** [comment]
Hey @PaprikaSteiger these are a bunch of random small changes. The reviewing is more to let you have a look in case something conflicts with what you are doing and less because I'm unsure – but in any case, two more eyes is good.

File: .coveragerc [hunk 1]
```diff
@@ -0,0 +1,17 @@
+[run]
+branch = True
+
+[report]
+exclude_lines =
+    # Manual way to exclude blocks
```
Comment (PaprikaSteiger) [inline .coveragerc:6]: @Anaphory Could I maybe get a general introduction to the purpose/function of this file? It looks like an interface for other software...

File: src/lexedata/change/clean_ids.py [hunk 2]
```diff
@@ -53,6 +53,47 @@ def clean_mapping(rows: t.Mapping[str, t.Mapping[str, str]]) -> t.Mapping[str, s
     return mapping
 
 
+def update_integer_ids(ds: pycldf.Dataset, table: csvw.metadata.Table):
```
Comment (PaprikaSteiger) [inline src/lexedata/change/clean_ids.py:56]: @Anaphory I think `mapping: t.Mapping[str, str]` might be missing as a function argument here

Comment (PaprikaSteiger) [inline src/lexedata/change/clean_ids.py:56]: @Anaphory I think `mapping: t.Mapping[str, str]` might be missing as a function argument or mapping.get shouldn't be here. When you call the function update_integer_ids() in the __main__ part, mapping is not yet defined.

Comment (Anaphory) [reply src/lexedata/change/clean_ids.py:56]: Probably true! I didn't think the integer IDs through very well, I just noticed a huge gap in the code.

Comment (Anaphory) [reply src/lexedata/change/clean_ids.py]: I fixed that step now. The functions are very similar, they could be generalized into one with two additional arguments, I think.

**Pull request merged**